### PR TITLE
show info for table and select option from database task.79011

### PR DESCRIPTION
**Pull request by Khang2561**
https://edu-redmine.sun-asterisk.vn/issues/79011
image

### CODE REVIEW

**Comment by Ngansan** [comment]
UI looks good!

**Comment by trucquynhCapy** [comment]
Code clear

**Comment by trietpm-0501** [comment]
lgtm

File: HotelManagerment/com.spring.sample/src/com/spring/sample/service/imp/RoomTypeServiceImp.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import com.spring.sample.entity.RoomType;
+import com.spring.sample.service.RoomTypeService;
+
+@Service
+public class RoomTypeServiceImp extends BaseServiceImpl implements RoomTypeService {
+	@Override
+	public Long loadCCountRecord() {
+		return roomTypeDAO.getCountRecord();
+	}
+
```
Comment (vanvtt-0952) [inline HotelManagerment/com.spring.sample/src/com/spring/sample/service/imp/RoomTypeServiceImp.java]: test xong thì xóa đi sếp

File: HotelManagerment/com.spring.sample/src/com/spring/sample/service/imp/RoomServiceImp.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@Service
+public class RoomServiceImp extends BaseServiceImpl implements RoomService {
+
+	private RoomServiceImp() {
+	}
+
+	@Override
+	public Long loadCCountRecord() {
+		return roomDAO.getCountRecord();
+	}
```
Comment (vanvtt-0952) [inline HotelManagerment/com.spring.sample/src/com/spring/sample/service/imp/RoomServiceImp.java]:
```suggestion
```

**Pull request merged**